tests/storage/assert_storage.rs

```diff
@@ -50,8 +52,12 @@ impl AssertionStorage {
     }
 
     pub fn update_with_key(&mut self, cluster: &mut Cluster<ServerCluster>, key: &str) {
-        cluster.must_get(key.as_bytes());
-        let region = cluster.get_region(key.as_bytes());
+        self.update_with_key_byte(cluster, key.as_bytes());
+    }
+
+    pub fn update_with_key_byte(&mut self, cluster: &mut Cluster<ServerCluster>, key: &[u8]) {
+        cluster.must_get(key);
```

Comment:
I prefer to use must_get directly  since query_leader need store_id and we can not get store id util use leader_of_region or in other similar way, that may change too much? @hhkbp2